docs/model_library/operational_water_management/index.rst

```diff
@@ -788,18 +824,20 @@ Completions reuse water is all water that meets completions pad demand, excludin
 
 
 
-**Piping Cost:** :math:`\forall (l,l) \in {PPA,\ldots ,CCA}, t \in T`
+**Piping Cost:** :math:`\forall \textcolor{blue}{l \in (L - O - K)}, \forall \textcolor{blue}{\tilde{l} \in (L - F)}, \forall \textcolor{blue}{(l,\tilde{l}) \in LLA}, \textcolor{blue}{t \in T}`
 
 Piping cost is the total volume of piped water multiplied by the cost for piping.
 
 .. math::
 
-    \textcolor{red}{C_{l,l,t}^{Piped}} = (\textcolor{red}{F_{l,l,t}^{Piped}}+\textcolor{red}{F_{l,l,t}^{Sourced}}) \cdot \textcolor{green}{\pi_{l,l}^{Pipeline}}
+    \textcolor{red}{C_{l,\tilde{l},t}^{Piped}}
```

Comment:
In the code (`PipingCostRule`), there is an `if` statement to write this constraint differently depending on whether or not the water being piped is sourced freshwater. The documentation should reflect that, in my opinion. The way the documentation is written here seems to be an attempt to combine both versions of the constraint into one, but I think it's possibly misleading to the reader. Better to just write each case for the constraint separately.